.github/workflows/code-health.yml

```
@@ -16,6 +16,26 @@ jobs:
   build:
     name: build
     runs-on: ubuntu-latest
+    env:
+        DOCKER_CLI_EXPERIMENTAL: enabled
     steps:
       - uses: actions/checkout@v4
+      - name: Set up docker with containerd snapshotter
+        uses: crazy-max/ghaction-setup-docker@v2
```

let's use gitSha instead of tag. Good article https://www.paloaltonetworks.com/blog/prisma-cloud/github-actions-worm-dependencies/#HowtoProtectYourWorkflowsandAssets
```suggestion
      - name: Set up docker with containerd snapshotter
        uses: crazy-max/ghaction-setup-docker@c2351bbd0bfab8cd65e684219ad8ea46a6d093f3
```

---

updated